docs/source/security/cors.mdx

```diff
@@ -223,8 +206,6 @@ The most important rule for whether or not a request is ["simple"](https://devel
 
 However, Apollo Server also handles [`GET` requests](../requests#get-requests). `GET` requests do not require a `Content-Type` header, so they can potentially be simple requests. So how can we ensure that we only execute `GET` requests that are _not_ simple requests? If we require the request to include an HTTP header that is never set automatically by the browser, then that is sufficient: requests that set HTTP headers other than the handful defined in the spec must be preflighted.
 
-Apollo Server 3.7 introduced a CSRF prevention feature, which can be enabled in versions of Apollo Server 3.7+ by passing `csrfPrevention: true` to the `ApolloServer` constructor.
-
 When this feature is enabled, Apollo Server only executes GraphQL operations if at least one of the following conditions is true:
```

Comment:
There's no antecedent for "this feature" any more.

Reply:
Great point!  Tweaked to:

`When CSRF protection is enabled, Apollo Server only executes GraphQL operations if at least one of the following conditions is true:`

Reply:
Hmm, we haven't even mentioned the fact that Apollo Server has a CSRF prevention feature (btw we usually say prevention not protection) since the very very beginning of the whole section...

Reply:
I see what you mean, how does this feel for tying these ideas together:

> By default, Apollo Server 4 has a CSRF prevention feature enabled. This means your server only executes GraphQL operations if at least one of the following conditions is true: